app/src/main/java/com/googleresearch/capturesync/SoftwareSyncController.java

```diff
@@ -204,10 +210,25 @@ private void setupSoftwareSync() {
 
                     if (settings != null) {
                         mState = State.SETTINGS_APPLICATION;
-                        mMainActivity.getApplicationInterface().getSoftwareSyncUtils().applyAndLockSettings(settings, () -> mState = State.IDLE);
+                        mMainActivity.getApplicationInterface().getSoftwareSyncUtils().applyAndLockSettings(settings, () -> {
+                            mMainActivity.getApplicationInterface().getSoftwareSyncUtils().prepareVideoRecording();
```

Comment:
Extract softwareSyncUtils to variable

Reply:
I think that there are some more places in file with usage of it, not only one string to fix, lets change every where